docs/appendices/release-notes/unreleased.rst

```diff
@@ -116,6 +116,10 @@ Changes
 
 - Updated Admin UI to 1.22.0, including an update with the new logo and colors.
 
+- Added ``SUBSTRING`` to non-reserved SQL keywords in order to support the
```

Comment:
Please check indentation, seems that also the above entry is not consistent.

I guess this is not considered breaking, only the other way around (non-reserved->reserved), right?

Reply:
> Please check indentation, seems that also the above entry is not consistent.

Hm looks fine to me. What's wrong here?

> I guess this is not considered breaking, only the other way around (non-reserved->reserved), right?

I'd say so as being more lenient should not break anything. Or do I miss something?

Reply:
Sorry, the github view shows it strangely as it's indented more:
![Screenshot_20220617_120955](https://user-images.githubusercontent.com/5058131/174267268-591509ba-449d-4dfe-9f73-a319c32ca69f.png)